README.md

```diff
@@ -121,6 +121,10 @@ can still be sent.
 
 The `Request` class is responsible for streaming the incoming request body
 and contains meta data which was parsed from the request headers.
+If the request body is chunked-encoded, the data will be decoded and emitted on the data event.
+The `Transfer-Encoding` header will be removed.
+
+Any other `Transfer-Encoding` than `chunked` will result in an error message.
```

Comment:
Not sure there this should be placed. "will result in an error message" where is this raised? See also chapter above?